tests/unit/disc/omegah/OmegahConMgr.cpp

```diff
@@ -91,6 +91,30 @@ void createTagFromClassification(Omega_h::Mesh& mesh, const int classId, const i
   mesh.add_tag(OMEGA_H_VERT, tagName, 1, lateralverts);
 }
 
+template<typename T>
+void checkOwnership(Omega_h::Mesh& mesh, T connMgr) {
+  const auto localElmIds = connMgr->getElementBlock("ignored");
+  auto conMgrVtxGids = Omega_h::HostRead(connMgr->getGlobalDofNumbering(OMEGA_H_VERT));
+  auto isVtxOwned = Omega_h::HostRead(mesh.owned(OMEGA_H_VERT));
+  std::map<Omega_h::GO, Omega_h::I8> vtxGidOwned;
+  for(int i = 0; i < conMgrVtxGids.size(); i++)
+    vtxGidOwned[conMgrVtxGids[i]] = isVtxOwned[i];
+  const auto partDim = connMgr->part_dim("ignored");
+  int dofsPerElm = connMgr->getConnectivitySize(0);
+  //this check only supports dofs at vertices
+  if(partDim == 1) REQUIRE(dofsPerElm == 2);
+  if(partDim == 2) REQUIRE(dofsPerElm == 3);
+  if(partDim == 3) REQUIRE(dofsPerElm == 4);
+  for( auto lid : localElmIds ) {
+    auto dofGids = connMgr->getConnectivity(lid);
+    auto dofOwned = connMgr->getOwnership(lid);
+    for(int i=0; i<dofsPerElm; i++) {
+      if(vtxGidOwned.at(dofGids[i]))
```

Comment:
To be safe, we could also check that `Albany::Ghosted == dofOwned[i]` in an else statement.

Reply:
Thank you.  I added this with [f1c1ee3](https://github.com/sandialabs/Albany/pull/977/commits/f1c1ee3647c77646a7ff0e9b0886ff66b826a808).